tests/test_web/test_module.py

```diff
@@ -389,6 +390,27 @@ async def test_subcomponent_notation_as_obj_attrs(display: DisplayFixture):
     assert len(form_label) == 1
 
 
+async def test_callable_prop_with_javacript(display: DisplayFixture):
```

Comment:
Another test is needed to see if `string_to_reactpy` works well with this implementation.